<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVVClass</class>
 <widget class="QMainWindow" name="QtVVClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>466</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>9000</width>
    <height>6000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>基于视频的车速车型检测</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(140,140,140);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="vvImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(219, 255, 254)</string>
    </property>
    <property name="text">
     <string>ShowRect</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(25,255,255);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pause">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(25,255,255);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartFind">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(25,255,255);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>显示
分析</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(25,255,255);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>660</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StopFind">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(25,255,255);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>停止
分析</string>
    </property>
   </widget>
   <zorder>backgroundLabel</zorder>
   <zorder>vvImg</zorder>
   <zorder>pushButton_Start</zorder>
   <zorder>pushButton_Pause</zorder>
   <zorder>pushButton_StartFind</zorder>
   <zorder>pushButton_Exit</zorder>
   <zorder>pushButton_StopFind</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(40,40,40);</string>
   </property>
  </widget>
  <action name="action_ConCam">
   <property name="text">
    <string>打开视频文件</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_ShowHis">
   <property name="text">
    <string>查看录像</string>
   </property>
  </action>
  <action name="action_Setting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>匹配设置</string>
   </property>
  </action>
  <action name="action_Faces">
   <property name="text">
    <string>匹配数据库</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>关于基于视频的车速车型检测</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtVV.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Exit</sender>
   <signal>clicked()</signal>
   <receiver>QtVVClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>QtVVClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
